CMakeLists.txt

```
@@ -25,5 +29,19 @@ if(BUILD_TESTING)
 endif()
 
 add_subdirectory(src/beman/exemplar)
-
 add_subdirectory(examples)
+
+write_basic_package_version_file(
+  ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_PROJECT_NAME}ConfigVersion.cmake
```

```suggestion
  ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_PROJECT_NAME}-config-version.cmake
```
Just because `lower.caseThenCamelCase.cmake` is not how anyone would choose to name a file. See [upstream docs](https://cmake.org/cmake/help/latest/manual/cmake-packages.7.html) on support for kebab-case filenames.